desktop-app/app/components/Addressinput/index.js

```diff
@@ -57,6 +58,25 @@ class AddressBar extends React.Component<Props> {
           onChange={e => this.setState({userTypedAddress: e.target.value})}
         />
         <div className={cx(styles.floatingOptionsContainer)}>
+          <div
```

Comment:
The icon in the screenshot looks a bit out of place with the others icons nearby. Shall we try reducing the size a bit?

Reply:
For the other icon you rely on a custom SVG apparently. I don't know where you took these icons but I would need to know to find a star SVG with the same stroke density.

Reply:
Those were custom icons put together by combining SVG and cross. Let's keep this on hold, I'll try to find one that matches here.

Reply:
I added small font size so it first a bit more but the stroke is still too wide.